webrisk/snippets/compute_threatlist_diff.py

```diff
@@ -0,0 +1,81 @@
+#  Copyright 2022 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# [START webrisk_compute_threatlist_diff]
+from google.cloud import webrisk_v1
+
+
+def compute_threatlist_diff(threat_type: webrisk_v1.ThreatType, version_token: bytes, max_diff_entries: int,
+                            max_database_entries: int, compression_type: webrisk_v1.CompressionType) -> None:
```

Comment:
Multiple parameters should be laid one per line for better readability.
```suggestion
def compute_threatlist_diff(
    threat_type: webrisk_v1.ThreatType,
    version_token: bytes,
    max_diff_entries: int,
    max_database_entries: int,
    compression_type: webrisk_v1.CompressionType,
) -> None:
```

Reply:
Done. Thanks.